delegatePlay/DelegateController.swift

```diff
@@ -8,19 +8,31 @@
 import UIKit
 
 // Accept the Protocol
-class DelegateController: UIViewController {
+class DelegateController: UIViewController, UseData {
+    func writeToLabel(info: String) {
+        <#code#>
```

Comment:
This function is the one declared in your protocol. Replace line 13 with line 22